std_msgs/Header header
GraphEdge[] edges
GraphNode[] nodes